src/vg.erl

```diff
@@ -1,11 +1,20 @@
 -module(vg).
 
--export([create_topic/1,
-         ensure_topic/1,
+%% client interface
+-export([ensure_topic/1,
          write/3, write/4,
          fetch/1, fetch/2, fetch/4,
          fetch/5]).
 
+%% ops interface.
+-export([
+         create_topic/1,
+         delete_topic/1,
+         describe_topic/1,
```

Comment:
I know that this one is missing, I just wanted to get the rest reviewed while I worked on the rest of the list.